impl/src/main/java/org/jboss/weld/config/ConfigurationKey.java

```diff
@@ -265,6 +265,14 @@
     @Description("If a non-empty string and development mode is enabled, the Probe data will be automatically exported after deployment validation. The value represents a path of the directory where to export the data file.")
     PROBE_EXPORT_DATA_AFTER_DEPLOYMENT("org.jboss.weld.probe.exportDataAfterDeployment", ""),
 
+    /**
+     * If true an alternative defined in beans.xml can be vetoed. If false an alternative defined in beans.xml will cause an exception to be thrown.
```

Comment:
I would slightly alter the second sentence to make it more readable:
 `If false, vetoing an alternative defined in beans.xml will cause an exception to be thrown.`